server/routes.js

```diff
@@ -0,0 +1,8 @@
+const routes = function (app, postManager, userManager) {
```

Comment:
This can be actually exported at the same time.
```suggestion
module.exports = function (app, postManager, userManager) {
```

Reply:
Perhaps, think about whether you could make these managers as a singleton.